pkg/webhook/cspc.go

```diff
@@ -527,11 +528,18 @@ func (p *PoolOperations) ValidateScaledown() (bool, string) {
 				break
 			}
 		}
+
+		ls := &metav1.LabelSelector{
```

Comment:
Can we move the initialization part inside the `if` block? (It will be more readable since we require this only if it is scale down case)

Reply:
Done. Thanks